internal/dbsqlc/query.sql

```diff
@@ -1,14 +1,6 @@
 -- name: JobCountByState :many
-SELECT
-  state,
-  count(*)
-FROM
-  river_job
-WHERE
-  queue IS NOT NULL AND
-  priority > 0 AND
-  scheduled_at IS NOT NULL AND
-  id IS NOT NULL
```

Comment:
I think this was originally in here in an attempt to nudge Postgres into taking advantage of the prioritized fetching index for this query, but tbh it may not have been the best way to do that. I assume you've ensured the index still gets used on large tables even after this change?

Reply:
Oh no, the index definitely does not get used (unless you reindex) — that's what's driving this whole change. A seq scan was reported in #106, and it turns out that the seq scan is mostly right because an index scan is approximately equivalent in cost, so Postgres decision is correct. The `NOT NULL`s are what already exist in the existing release and it happens either way.